public/content/documentation/native/controls/captcha.md

```diff
@@ -0,0 +1,288 @@
+    3. Each element in audio challenge receives focus by swipe or touch. User is confined inside a modal, if presented
+- Use the device's default focus functionality. 
+- Consider how focus should be managed between child elements and their parent views.
+- Initial focus on a screen should land in a logical place, such as back button, screen title, first text field, or first heading.
+
+- **UIKit**
+    - If VoiceOver is not reaching a particular element, set the element's `isAccessibilityElement` to `true`
+        - **Note:** You may need to adjust the programmatic name, role, state, and/or value after doing this, as this action may overwrite previously configured accessibility.
+    - Use `accessibilityViewIsModal` to contain the screen reader focus inside the modal.
+    - To move screen reader focus to newly revealed content, use `UIAccessibility.post(notification:argument:)` that takes in `.screenChanged` and the newly revealed content as the parameter arguments.
+    - To NOT move focus, but dynamically announce new content: use `UIAccessibility.post(notification:argument:)` that takes in `.announcement` and the announcement text as the parameter arguments.
+    - `UIAccessibilityContainer` protocol: Have a table of elements that defines the reading order of the elements.
+
+- **SwiftUI**
+    - For general focus management that impacts both screen readers and non-screen readers, use the property wrapper `@FocusState` to assign an identity of a focus state.
+        - Use the property wrapper `@FocusState` in conjunction with the view modifier `focused(_:)` to assign focus on a view with `@FocusState` as the source of truth.
+        - Use the property wrapper `@FocusState`in conjunction with the view modifier `focused(_:equals:)` to assign focus on a view, when the view is equal to a specific value.
+    - If necessary, use property wrapper `@AccessibilityFocusState` to assign identifiers to specific views to manually shift focus from one view to another as the user interacts with the screen with VoiceOver on.
+
+## Android Developer Notes
+
+- There are several experiences for Captcha:  
+    1. A non-interactive image in a table row, where the whole table row acts like a button.  
+    2. A single interactive element, like an image button
+    3. An audio challenge as an equivalent experience to the image challenge
+- When selecting the type of Captcha, do not limit the options to only <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html#dfn-cognitive-function-test"> cognitive function tests</a> (e.g. remembering or transcribing a word, or recognizing a picture the website provided).  When possible, leverage <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-enhanced.html#examples"> recommended techniques recommended by the W3C</a> such as email link authentication, 3rd party  authentication, or 2 factor authentication.
```

Comment:
an aria-label is being added when I view in the browser but I don't see it here ... the aria-label seems a bit lengthy with the full url spelled out. Is this something being done at a template-level or am I missing something here?

Also, I'm trying to figure out why these links don't have an underline though it doesn't seem to be anything you're doing.

Reply:
Ah, for the second issue I noted there's a style `.MagentaA11y__content-details a` that is overriding the normal link underline.

Reply:
@silverli - do you have any insight to the aria-label situation happening? 

@danbivins - is there any update on the underlining not populating that I missed Thurs/Fri?

Reply:
This is a global issue that Lindsay will fix.